base/applications/regedit/main.c

```diff
@@ -217,6 +218,29 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
         default:
             break;
     }
+
+    /* Prevent multiple instances of application */
+    hMutex = CreateMutexW(NULL, FALSE, L"Regedit");
+
+    if (hMutex)
+    {
+        if (GetLastError() == ERROR_ALREADY_EXISTS)
+        {
+            /* The application's instance is already running */
+            HWND hCurrentWnd = FindWindowW(szFrameClass, szTitle);
```

Comment:
What if I start a russian-translated ROS regedit on an english Windows installation that is also running an english-translated regedit?